### PR TITLE
Created C# docs page

### PR DESCRIPTION
**Pull request by sandi-inforit**
### Features and Changes

Just added a new page with the documentation for the C# package.
Took it from the package repository and added a few tweaks to have it in line with the other documentation pages

### CODE REVIEW

File: docs/docs/lib/csharp.mdx [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+title: C# SDK
+description: GrowthBook SDK for C#
+sidebar_label: C#
+slug: C#
```
Comment (jdorn) [inline docs/docs/lib/csharp.mdx:5]: This URL slug is invalid

```suggestion
slug: c-sharp
```

File: docs/docs/lib/csharp.mdx [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
+title: C# SDK
+description: GrowthBook SDK for C#
+sidebar_label: C#
```
Comment (jdorn) [inline docs/docs/lib/csharp.mdx:4]: To make this show up in the sidebar, you also have to edit `sidebars.js` and add this to the array on line 100.

In addition, on `docs/lib/index.mdx`, there is a link to the C# github repo.  We should probably change that to point to this new page instead.

File: docs/docs/lib/csharp.mdx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+# C#
+
+View the full documentation on [GitHub](https://github.com/growthbook/growthbook-c-sharp).
+
+[![nuget badge for growthbook](https://img.shields.io/nuget/v/growthbook-c-sharp?style=flat-square)](https://www.nuget.org/packages/growthbook-c-sharp)
+
+## Installation
+
+```csharp
```
Comment (jdorn) [inline docs/docs/lib/csharp.mdx:16]: To make this highlighting work, we need to add `"csharp"` to the `additionalLanguages` array in `docusaurus.config.js` on line 136

**Review by jdorn — CHANGES_REQUESTED** [review]
Thanks for contributing this!  I added a couple small comments.  You can test the docs locally by doing:

```
cd docs
yarn install
yarn dev
```

Then, go to http://localhost:3200 to view once it's done building

File: docs/docs/lib/csharp.mdx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+Down below you'll find two basic examples on how the package works.
+
+### Basic
+
+1. Declare features
+
+    ```csharp
+
+    var staticFeatures = new Dictionary<string, Feature>
+        {
```
Comment (pascal89) [inline docs/docs/lib/csharp.mdx:33]: Your indenting is off, should be like this:
```cshap
    var staticFeatures = new Dictionary<string, Feature>
    {
        { "firstFeature", new Feature { DefaultValue = true }},
        { "secondFeature", new Feature { DefaultValue = false }}
    };
```

**Review by pascal89 — COMMENTED** [review]
I'd suggest to apply the requested changes. Also I'd like to suggest to read the [CODE_OF_CONDUCT.md](https://github.com/growthbook/growthbook/blob/main/CODE_OF_CONDUCT.md) and the [CONTRIBUTING.md](https://github.com/growthbook/growthbook/blob/main/CONTRIBUTING.md) before you contribute to an open source project ;-) 

@maintainer this PR would close this issue: https://github.com/growthbook/growthbook-c-sharp/issues/2

**Review by Auz — APPROVED** [review]

**Pull request merged**